docs/registering-ticket-sales-group.md

```diff
@@ -0,0 +1,155 @@
+# Registering ticket sales for group passes
+
+This mini-guide illustrates how to register UiTPAS discounted ticket sales for a group pass. It builds upon the [Registering ticket sales guide](/docs/registering-ticket-sales-group.md) which you need to read first to learn more about authentication, the work flow and registering events.
+
+> ##### What is a group pass?
+>
+> A group pass is very similar to a regular passholder, but it is not bound to one specific person, but to an organisation for example. These passes can be used to buy multiple tickets for the same discounted price, instead of just one. It's also identified by an UiTPAS number. The number of tickets that can be registered at a specific UiTPAS tariff is usually preset when the group pass is issued.
+
+This mini-guide hooks into the regular flow when an end-user entered the UiTPAS number of a group pass instead of a regular passholder.
+
+### 1. Determine possible UiTPAS tariffs
+
+Using the event id, the UiTPAS number and the regular price of your event, you can [request possible UiTPAS tariffs](/reference/UiTPAS.v2.json/paths/~1tariffs/get).
+
+Your application doesn't (have to) know if the given UiTPAS number is a regular passholder or a group pass.
+
+Example request:
+
+```http
+GET /tariffs/?eventId=5a0967f9-cc06-4c3c-9206-30481a767434&uitpasNumber=0900010630813&regularPrice=10 HTTP/1.1
+Host: https://api.uitpas.be
+Authorization: Bearer YOUR_CLIENT_ACCESS_TOKEN'
```

Comment:
Maybe the guide is more futureproof if we replace `YOUR_CLIENT_ACCESS_TOKEN` with a more generic `YOUR_ACCESS_TOKEN`? I know most will use client access tokens, but I'm not sure if we need to go this specific here because usually an integrator will only be able to get one or the other and they will probably look at it as "a" token after they get it?